tests/x11/addon_products_via_SCC_yast2.pm

```diff
@@ -0,0 +1,52 @@
+# SUSE's openQA tests
+#
+# Copyright 2012-2019 SUSE LLC
```

Comment:
copyright should be only 2022, check the other files just in case would need update.